src/controllers/apps/chat-app/message.controllers.js

```diff
@@ -154,5 +158,128 @@ const sendMessage = asyncHandler(async (req, res) => {
     .status(201)
     .json(new ApiResponse(201, receivedMessage, "Message saved successfully"));
 });
+const deleteMessage = asyncHandler(async (req, res) => {
+  //controller to delete chat messages and attachments
 
-export { getAllMessages, sendMessage };
+  const { chatId, messageId, attachmentId } = req.params;
+
+  //Find the chat based on chatId
+
+  const chat = await Chat.findOne({
+    _id: new mongoose.Types.ObjectId(chatId),
```

Comment:
add check if logged in user is part of that chat so add more condition to check of logged in user exists in the `participants` array